frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class CreateTripWidget extends StatefulWidget {
+  CreateTripWidget({Key key}) : super(key: key);
+  @override
+  CreateTripState createState() => CreateTripState();
+}
+
+class CreateTripState extends State<CreateTripWidget> {
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Form(
+      key: _formKey,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          TextFormField(
```

Comment:
I need to look more into `Form` and `TextFormField` widgets, but I think we just want `TextField` widgets since we are not going to be using HTML like form submission.